implementations/rust/ockam/ockam_command/src/zone/repl.rs

```diff
@@ -91,6 +93,13 @@ impl ReplCommand {
                 } else {
                     "the portal"
                 };
+                if let Some(logs_address) = logs_address {
+                    opts.terminal.write_line(fmt_log!(
+                        "The zone logs are accessible at http://localhost:{}/",
```

Comment:
For consistency, we should use color_primary for the `http://localhost:{}` url

Reply:
Do we want to show this url only when there is no repl?

Reply:
Fixed